build_web_compilers/build.yaml

```diff
@@ -11,20 +11,31 @@ builders:
         - .ddc.js.map
     is_optional: True
     auto_apply: all_packages
-    required_inputs: [".dart"]
+    required_inputs:
+      - ".dart"
```

Comment:
[nit] inconsistent quoting with line 27

Out of curiosity - is there a reason to prefer this style over the more compact version?

Reply:
Just for consistency, we have other lists that would end up wrapping. We could change everything to this style too, but it feels a bit weird in yaml to me.